addons/google_calendar/tests/test_sync_odoo2google.py

```diff
@@ -23,6 +23,16 @@ def setUp(self):
         self.google_service = GoogleCalendarService(self.env['google.service'])
         # Make sure this test will work for the next 30 years
         self.env['ir.config_parameter'].set_param('google_calendar.sync.range_days', 10000)
+        self.user1 = self.env['res.users'].create({
+            'name': 'user1',
+            'login': 'user1',
+            'email': 'user1@odoo.com',
+        })
+        self.private_partner = self.env['res.partner'].create({
+            'name': 'Private Contact',
+            'email': 'private_email@example.com',
+            'type': 'private',
+        })
```

Comment:
Could you please move the two users creation to inside your test `test_partner_type_change`?
As it is a specific case they won't be needed in other tests (therefore we won't need to set them in `setUp`).
```suggestion
```

Reply:
Done